### ISSUE TITLE
autoplay flag

### ISSUE THREAD

**Issue by mahemoff** [podlove/podlove-specifications#13]
I think it could be useful to extend the deep-linking spec to include an autoplay flag in the URL, as with YouTube.

**Comment by line-o** [comment]
Good point. Especially, since the project moves on towards embeddability.
Youtube uses `?autoplay=1` to determine, if an embedded player should start on frame load.

**Comment by timpritlove** [comment]
Yes, this might be an optional parameter. The spec currently does not say anything what the player should do although the Podlove Web Player currently actually does autoplay.
